prowler/lib/scan/scan.py

```diff
@@ -19,7 +20,7 @@ class Scan:
     _progress: float = 0.0
     _findings: list = []
 
-    def __init__(self, provider: Provider, checks_to_execute: list[str]):
+    def __init__(self, provider: Provider, checks_to_execute: list[str] = []):
```

Comment:
```suggestion
    def __init__(self, provider: Provider, checks_to_execute: list[str] = None):
```

rationale -> https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

Reply:
Ok! Thanks